test/helpers/mock_resource.rb

```diff
@@ -1,6 +1,9 @@
 # frozen_string_literal: true
 
 MockResource = Struct.new(:id, :hits_to_complete, :status) do
+  ::SYNC_DEPENDENCIES = []
```

Comment:
Is this just due to some strange behavior of how Struct works?

Reply:
I guess so yep. I can tinker with it to see how many other ways I can get it to work (like `self::SYNC_DEPENDENCIES`) but just putting the constant there by itself is no dice in this case.

Reply:
Hmm it didn't break the tests when I tried it this time around so sticking with `self::` 👍